bots/images/scripts/rhel.setup

```diff
@@ -51,10 +51,11 @@ if [ ! -f "$SKIP_REPO_FLAG" ]; then
         yum-config-manager --enable rhel-7-server-extras-rpms
 
     elif [ "$IMAGE" = "rhel-7-7" ]; then
+        # HACK: No Extras 7.7 nightly repo yet
 cat <<EOF > /etc/yum.repos.d/nightly.repo
-[EXTRAS-7.7-LATEST]
+[EXTRAS-7.6-LATEST]
 name=rhel-extras-compose
-baseurl=http://download.devel.redhat.com/nightly/EXTRAS-RHEL-7.7/latest-EXTRAS-7.7-RHEL-7/compose/Server/x86_64/os/
+baseurl=http://download.devel.redhat.com/rhel-7/rel-eng/EXTRAS-7/latest-EXTRAS-7.6-RHEL-7/compose/Server/x86_64/os/
```

Comment:
Can you please keep the "future" line as a comment, so that it's easy for us to switch once it exists?